website/client/src/components/auth/register.vue

```diff
@@ -0,0 +1,342 @@
+<template>
+  <div
```

Comment:
Can this and other forms use a `form` element, we should have forms being `form` elements to maintain maximum accessibility and make it possible to submit by pressing Enter on any input inside the form itself

Reply:
got it~ thx for point this out